### ISSUE TITLE
[MU4 Issue] Accessibility: Narrator does not read accidentals in key signatures or text

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#12898]
**Describe the bug**
When using the screen reader Narrator, accidentals are read on notes, but not in key signatures or text.

**To Reproduce**
1. Start Narrator
2. Create score in F# major
3. Click key signature
4. See error - Narrator reads "F major"
5. Add chord symbol F#
6. Navigate away and back to force Narrator to read it
7. See error - Narrator reads "Chord Symbol F"

**Expected behavior**
Narrator should read accidentals.

**Platform information**
 - OS: Windows

**Additional context**
The issue is apparently a Narrator bug (?), but we should work around it as we do for notes.

**Comment by zacjansheski** [comment]
Narrator also does not read the sharp or flat signs when choosing a key signature in the new score window.  

NVDA does announce the sharp/flat.

**Comment by DmitryArefiev** [comment]
Confirm on my side too. It's fine with Orca and Voice Over (VO reads "number" and "b" instead of sharp and flat, but it's better than nothing)

Added video for description

**Comment by jmwmusic** [comment]
I am told this is *NOT* a MuseScore issue. It is a shortcoming in Narrator and should be referred to Microsoft.
MuseScore uses the official Unicode characters for flat and sharp, which Narrator cannot read, therefore they are skipped.
You have the same problem in Narrator with many other Unicode characters, including ✓ ♫ ∨ ♅ … ′  and many others.
I also did not find a punctuation level control in the Narrator settings. If this is firmly a Narrator problem, not a MuseScore problem then this issue might be closed.
closed.
